### PR TITLE
Feature/glacier storage

### PR DESCRIPTION
**Pull request by afquinterog**

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -38,8 +38,12 @@ echo "-----> backing up to s3 ... "
 #compress actual code
 tar -cvzPf /tmp/backup.tgz $BUILD_DIR > /dev/null
```
Comment (epricer) [inline bin/compile]: you're using the verbose output (v flag), but then just dumping stdout it to /dev/null.

File: bin/compile [hunk 2]
```diff
@@ -38,8 +38,12 @@ echo "-----> backing up to s3 ... "
...
 
-#send code to s3
-/tmp/aws/bin/aws s3 cp /tmp/backup.tgz s3://$VCO_BACKUPS_BUCKET_NAME/$VCO_BACKUPS_APP_NAME/$dt.tgz --sse
+#Save data on Glacier vault
+echo "-----> Archive description: $VCO_BACKUPS_APP_NAME-$dt " 
+/tmp/aws/bin/aws glacier upload-archive --archive-description $VCO_BACKUPS_APP_NAME-$dt \
```
Comment (epricer) [inline bin/compile]: How does this work? It doesn't reference the tgz file
Also, what is the backslash at the end of the line for?

File: bin/compile [hunk 3]
```diff
@@ -38,8 +38,12 @@ echo "-----> backing up to s3 ... "
...
+																			  --account-id -  \
+																			  --vault-name $VCO_BACKUPS_VAULT_NAME \
+																			  --body /tmp/backup.tgz
```
Comment (epricer) [inline bin/compile]: what are all these blank lines for?

File: bin/compile (hunk 2 above)
Comment (afquinterog) [reply bin/compile]: Improved the format, a little.

**Pull request merged**